lib/Controller/ChatController.php

```diff
@@ -689,37 +692,58 @@ public function deleteMessage(int $messageId): DataResponse {
 	#[RequireModeratorOrNoLobby]
 	#[RequireLoggedInParticipant]
 	#[UserRateLimit(limit: 60, period: 3600)]
-	public function remindLater(int $messageId, int $timestamp): DataResponse {
+	public function setReminder(int $messageId, int $timestamp): DataResponse {
 		try {
-			$message = $this->chatManager->getComment($this->room, (string) $messageId);
+			$this->chatManager->getComment($this->room, (string) $messageId);
```

Comment:
No `exists` method?
Depending of the implementation, it could be faster than a `get`